pkgs/applications/networking/cluster/sonobuoy/default.nix

```diff
@@ -5,7 +5,7 @@ let rev = "551bf68c694927839c3add25a2512f880902ee9b";
 in
 buildGoModule rec {
   pname = "sonobuoy";
-  version = "0.56.10"; # Do not forget to update `rev` above
+  version = "0.56.12"; # Do not forget to update `rev` above
```

Comment:
We have to change the `rev` above. Can someone with push rights to the PR fix that or do we have to create a new one?

Reply:
We really need a custom update script that can handle this :upside_down_face: